crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -451,12 +450,8 @@ impl<'w, 's, 'a> EntityCommands<'w, 's, 'a> {
     /// }
     /// # example_system.system();
     /// ```
-    pub fn insert(&mut self, component: impl Component) -> &mut Self {
-        self.commands.add(Insert {
-            entity: self.entity,
-            component,
-        });
-        self
+    pub fn insert(&mut self, bundle: impl Bundle) -> &mut Self {
+        self.insert_bundle(bundle)
```

Comment:
> Because I want this pr to be reviewable, not just a mess of `insert_bundle`->`insert` renamings
> If this gets approved we can change them over in a follow up pr